### DISCUSSION TITLE
Custom selection

### DISCUSSION THREAD

**Discussion by Fee0** [glideapps/glide-data-grid#250] [Q&A] [answered]
Hi,

I thought about using glide to implement an online hex editor. For that I would need some custom selection mode as the data order goes like a text from left to right and from top to bottom. See the image as an example where you select "11" and drag the mouse down to "12" which causes all the row data in between to be selected. 

![grafik](https://user-images.githubusercontent.com/11669187/161820323-fd1e10c3-3df3-4b11-ba58-ff6a6b1a93a7.png)

Is something like that supported by glide?

**Comment by jassmith** [comment, marked as answer]
check out highlight regions in the 4.0.0 release. https://github.com/glideapps/glide-data-grid/releases/tag/v4.0.0

**Reply by Fee0** [reply]
Thank you @jassmith that looks promising I gonna play around with that.
By the way I tried the hightlight demo at [https://glideapps.github.io/glide-data-grid/?path=/story/glide-data-grid-dataeditor-demos--highlight-cells](url) and it cannot handle to select anything beyond column 88.

**Reply by jassmith** [reply]
Oh lol yeah probably should have not made it dumbly set the highlight region without checking the outer bounds. Will fix.

Edit - Just to be clear I consider this a coding error in the demo more than the actual data grid. Highlight regions should never include cells/regions outside the actual specified width/height of the data.
